src/main/python/janusgraph_python/serializer/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains all the Serializer and Deserializer classes needed for converting between
```

Comment:
@FlorianHockmann : This includes will include the serializer and deserializers for different types. So tomorrow we add `Point`, `Circle` their Serializer and Deserializer will go into this module. While `structure.io.graphson` will always contain 2 classes namely `GraphsonReader & GraphsonWriter` which registers the default set of serializers and deserializers as part of that particular release version

Reply:
But these serializers are specific to GraphSON and I don't see a reason to have a serializer or deserializer for a type but not include it already by default in `GraphsonReader` / `GraphsonWriter`. Once we add a serializer we will register it in the writer.
So, what's the point in putting those in different modules?

Reply:
So, how would you like to structure it then @FlorianHockmann ?
Something like single module (root) as `structure.io.graphson` and it having 2 sub modules as `serializers` and `builder` ? 

Then `structure.io.graphson.serializers` will contains classes like `relation_identifier_serilaizer` `relation_identifier_deserializer` ? If we add `point_serialiazer` and `point_deseializer` it goes into `structure.io.graphson.serializers` always?
As for contents of `structure.io.graphson.builder` it will always contain 2 classes namely `graphson_reader_builder` and `graphson_writer_builder` is that correct? 

Or do you have something else in mind?

Reply:
I would just put everything in a module `structure.io.graphson` without any more nesting. Why shouldn't the builder and the serializers be part of the same module? They belong together and cannot really be used without each other. So, why not have them in one module?